wormhole-connect/README.md

```diff
@@ -82,3 +82,36 @@ class WormholeConnect extends React.Component {
   }
 }
 ```
+
+## Togle Hamburguer Menu
+
+by setting `showHamburgerMenu` to **false**, you can toggle hamburguer menu off
+and that makes the links to be placed at bottom
```

Comment:
By setting the `showHamburgerMenu` option to **false**, you can deactivate the hamburger menu, causing the links to be positioned at the bottom.

Reply:
thanks, updated